spec/01-unit/19-hybrid/04-validate_deltas_spec.lua

```diff
@@ -131,10 +131,39 @@ describe("[delta validations]",function()
 
     local deltas = declarative.export_config_sync()
 
+    ngx.log(ngx.ERR, "xxxxxx ", require("inspect")(deltas))
```

Comment:
debug log?

Reply:
I'm debugging the test case ,will delete it later

Reply:
fixed